domainlab/mk_exp.py

```diff
@@ -10,10 +10,24 @@
 
 
 
-def mk_exp(task, model, trainer, test_domain, batchsize):
+def mk_exp(task, model, trainer:str, test_domain:str, batchsize:int):
     """
-    test_domain must be a single string
+    Creates a custom experiment. The user can specify the input parameters.
+
+    Input Parameters:
+        - task: create a task to a custom dataset by importing "mk_task_dset" function from
+        "domainlab.tasks.task_dset". For more explanation on the input params refer to the
+        documentation found in "domainlab.tasks.task_dset.py".
+        - model: create a model "myModel" by importing "mk_myModel" function from
```

Comment:
change myModel to "[NameOfTheModel]",   usually in computer science, one can use [] as placeholder